starknet-core/src/types/transaction.rs

```diff
@@ -26,8 +28,8 @@ pub struct TransactionWithStatus {
     pub transaction: Option<Transaction>,
     pub status: TransactionStatusType,
     #[serde(default)]
-    #[serde(deserialize_with = "deserialize_pending_block_hash")]
-    pub block_hash: Option<H256>,
+    #[serde(deserialize_with = "pending_block_hash_de")]
```

Comment:
The `#[serde]`s here could be merge into one line.

Reply:
Yes ser